test/dynamo/test_sys.py

```diff
@@ -0,0 +1,109 @@
+# Owner(s): ["module: dynamo"]
+import sys
+import unittest
+
+import torch
+import torch._dynamo.test_case
+
+from .utils import make_dynamo_test
+
+
+class SysTests(torch._dynamo.test_case.TestCase):
+    def test_exc_info(self):
+        @torch.compile(backend="eager", fullgraph=True)
+        def fn(t):
+            try:
+                raise ValueError
+            except Exception:
+                typ, _, _ = sys.exc_info()
+                if typ is ValueError:
+                    return t.sin()
+                else:
+                    return t.cos()
+
+        t = torch.randn(2)
+        y = fn(t)
+        self.assertEqual(y, t.sin())
+
+
+class CPythonActiveExceptionTests(torch._dynamo.test_case.TestCase):
```

Comment:
Missing link to https://github.com/python/cpython/blob/v3.13.1/Lib/test/test_sys.py